bundles/org.openhab.binding.dirigera/README.md

```diff
@@ -0,0 +1,781 @@
+
+**OTA Mappings**
+
+Mappings for `ota-status`
+
+- 0 : Up to date
+- 1 : Update available
+
+Mappings for `ota-state`
+
+- 0 : Ready to check
+- 1 : Check in progress
+- 2 : Ready to download
+- 3 : Download in progress
+- 4 : Update in progress
+- 5 : Update failed
+- 6 : Ready to update
+- 7 : Check failed
+- 8 : Download failed
+- 9 : Update complete
+- 10 : Battery check failed
+
+#### Links and Candidates
+
+Devices can be connected directly e.g. sensors or controllers with lights, plugs, blinds or speakers.
+It's detected during runtime if a device is capable to support links _and_ if devices are available in your system to support this connection.
+The channels are declered advanced and can be used for setup procedure.
+
+| Channel               | Type                  | Read/Write | Description                                      | Advanced |
+|-----------------------|-----------------------|------------|--------------------------------------------------|----------|
+| `links`               | String                | RW         | Linked controllers and sensors                   |    X     |
+| `link-candidates`     | String                | RW         | Candidates which can be linked                   |    X     |
+
+<img align="right" width="300" src="doc/link-candidates.png">
+
+Several devices can be linked together like
+
+- [Light Controller](#light-controller) and [Motion Sensors](#motion-sensor) to [Plugs](#power-plugs) and [Lights](#lights)
+- [Blind Controller](#blind-controller) to [Blinds](#blinds)
+- [Sound Controller](#sound-controller) to [Speakers](#speaker)
+
+Established links are shown in channel `links`.
+The linked devices can be clicked in the UI and the link will be removed.
+
+Possible candidates to be linked are shown in channel `link-candidates`.
+If a candidate is clicked in the UI the link will be established.
+
+Candidates and links marked with `(!)` are not present in openHAB environment so no handler is created yet.
+In this case it's possible not all links are shown in the UI, but the present ones shall work.
+
+#### Other Channels
+
+| Channel               | Type              | Read/Write | Description                                  | Advanced |
+|-----------------------|-------------------|------------|----------------------------------------------|----------|
+| `startup`             | Number            | RW         | Startup behavior after power cutoff          |          |
+| `custom-name`         | String            | RW         | Name given from IKEA home smart              |          |
+
+`startup` defines how the device shall behave after a power cutoff.
+If there's a dedicated hardwired light switch which cuts power towards the bulb it makes sense to switch them on every time the switch is pressed.
+But it's also possible to recover the last state.
+
+Mappings for `startup`
+
+- 0 : Previous
+- 1 : On
+- 2 : Off
+- 3 : Switch
```

Comment:
See earlier comment, better to have option lists.

Reply:
[Same comment](https://github.com/openhab/openhab-addons/pull/17719#discussion_r1882897432)